mne/preprocessing/ica.py

```diff
@@ -1486,21 +1488,33 @@ def _pick_sources(self, data, include, exclude):
         logger.info('Zeroing out %i ICA components'
                     % (n_components - len(sel_keep)))
 
-        unmixing = np.eye(_n_pca_comp)
-        unmixing[:n_components, :n_components] = self.unmixing_matrix_
-        unmixing = np.dot(unmixing, self.pca_components_[:_n_pca_comp])
+        if (self.method == 'imported_eeglab' and
+           np.diff(self.unmixing_matrix_.shape)[0]):
+            # When \PCA reduction is used in EEGLAB, runica returns
+            # weights= weights*sphere*eigenvectors(:,1:ncomps)';
+            # sphere = eye(urchans);
+            # such that we cannot separate the unmixing matrix and
+            # the PCA components. For that reason, the composite
+            # unmixing matrix takes a rectangular shape.
```

Comment:
this makes the current docstring not correct all the time. Can you add a sentence to say this in the docstring that documents attributes?

Reply:
I've adjusted the docstrings accordingly.